src/app/utils/plug-descriptions.ts

```diff
@@ -203,7 +183,7 @@ function getPerkDescriptions(
   Most plugs use the description field to describe their functionality.
 
   Some plugs (e.g. armor mods) store their functionality in their perk descriptions and use the description
-  field for auxiliary info like requirements and caveats. For these plugs, we want to prioritise strings in the
+  field for auxiliary info like requirements and caveats. For these plugs, we want to prioritize strings in the
```

Comment:
Thanks for tidying these up 😅

Reply:
I was looking around and the "Code spell checker" extension highlighted that so I fixed